recipes/mpir/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "2.7.2" %}
+
+package:
+  name: mpir
+  version: {{ version }}
+
+source:
+  fn: mpir-{{ version }}.tar.bz2
+  url: http://mpir.org/mpir-{{ version }}.tar.bz2
+
+build:
+  number: 0
+  features:
+    - vc14    [win]
+
+requirements:
+  build:
+    - m4 [unix]
+    - python
+
+test:
+  files:
+    - test.c
+
+about:
+  home: http://mpir.org/
+  license: LGPL 3
+  summary: Multiple Precision Integers and Rationals.
+
+extra:
+  recipe-maintainers:
+    - asmeurer
+    - isuruf
+    - jakirkham
```

Comment:
Always make sure to ask anyone you add. Preferably ask first, but if someone already expressed interest you can add them. Just make sure to check that it is ok.

Given I am on `gmp`, I am ok with this, but tend to spend most of my time in Mac or Linux dev environments. So, I don't know how handy I will be with the Windows stuff.